lib/plds/application.ex

```diff
@@ -15,6 +21,55 @@ defmodule PLDS.Application do
     Supervisor.start_link(children, opts)
   end
 
+  defp ensure_distribution! do
+    unless Node.alive?() do
+      case System.cmd("epmd", ["-daemon"]) do
+        {_, 0} ->
+          :ok
+
+        _ ->
+          PLDS.Utils.abort!("""
+          could not start epmd (Erlang Port Mapper Driver). PLDS uses epmd to \
+          talk to different runtimes. You may have to start epmd explicitly by calling:
+
+              epmd -daemon
+
+          Or by calling:
+
+              elixir --sname test -e "IO.puts node()"
+
+          Then you can try booting PLDS again
+          """)
+      end
+
+      {type, name} = get_node_type_and_name()
+
+      # TODO: Should I start it hidden? If so, how?
```

Comment:
The escript configuration in mix.exs allows you to set emu_args. Pass the -hidden flag there. :)